### PR TITLE
Fixed a few issues

### PR DESCRIPTION
**Pull request by yuezh**
1. Fixed init script for SUSE
2. Merged RP from SUSE
3. Added timeout to http request.

### CODE REVIEW

File: init/suse/waagent [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#
+# Note that starting an already running service, stopping
+# or restarting a not-running service as well as the restart
+# with force-reload (in case signalling is not supported) are
+# considered a success.
+
+
+case "$1" in
+    start)
+        echo -n "Starting WindowsAzureLinuxAgent"
```
Comment (thomas1206) [inline init/suse/waagent]: Change to MicrosoftAzure?

File: init/suse/waagent [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        ## Start daemon with startproc(8). If this fails
+        ## the echo return value is set appropriate.
+        startproc -f ${PYTHON} ${WAZD_BIN} -start
+        rc_status -v
+        ;;
+    stop)
+        echo -n "Shutting down WindowsAzureLinuxAgent"
```
Comment (thomas1206) [inline init/suse/waagent]: Change to MicrosoftAzure

File: init/suse/waagent [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        $0 stop
+        sleep 1
+        $0 start
+        rc_status
+        ;;
+    force-reload|reload)
+        rc_status
+        ;;
+    status)
+        echo -n "Checking for service WindowsAzureLinuxAgent "
```
Comment (thomas1206) [inline init/suse/waagent]: Same as above

**Comment by lizzha** [comment]
Looks good to me

**Pull request merged**